.github/STALE-BOT.md

```diff
@@ -0,0 +1,36 @@
+# Stale bot information
+
+- Thank you so much for your contribution!
+- If you would like the bot to remove the label, simply leave a comment.
+- _How to find the right people to ping?_ &rarr; [`git blame`](https://git-scm.com/docs/git-blame) to the rescue or GitHub's history button!
+- You can always ask for help on [our Discourse Forum](https://discourse.nixos.org/) or on the [#nixos' IRC channel](https://webchat.freenode.net/#nixos).
+
+## PR author suggestions
+
+1. If it is unfinished and you plan on finishing it, please mark it as a draft.
+2. If you want to abandon this work, please consider closing it with a short comment. This might encourage somebody else to pick up your work.
+3. If you want to get things rolling again, first rebase the PR and address all valid comments.
+4. Then, if you need a review to push things forward, [try this Discourse thread](https://discourse.nixos.org/t/prs-in-distress/3604).
+5. If all you need is a merge, check the git history to find and [request reviews](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/requesting-a-pull-request-review) from people who usually merge related contributions.
+
+## Issue author suggestions
+
+1. If it is resolved (either for you personally, or in general), please consider closing it.
+2. If this might still be an issue, but you are not interested in promoting its resolution, please consider closing it while encouraging others to take over and reopen an issue if they care enough.
+3. If you still have interest in resolving it, try to ping somebody who you believe might have an interest in the topic. Consider discussing the problem in [our Discourse Forum](https://discourse.nixos.org/).
+4. As with all open source projects, your best option is to submit a Pull Request that addresses this issue. We :heart: this attitude!
+
+**Memorandum on closing issues**
+
+- Closed issues are still around, and people _do_ search closed issues, so nothing is lost.
```

Comment:
```suggestion
Don't be afraid to close an issue that holds valuable information. Closed issues stay in the system for people to search, read, cross-reference, or even reopen--nothing is lost! Closing obsolete issues is an important way to help maintainers focus their time and effort.
```
The third bullet in this list felt a bit thin to me, so I started off wanting to cut it, but then the whole list started to feel a little thin, so I found myself rewriting it as a paragraph... Does this still capture the sentiment you were aiming for?

Reply:
I like it.